stable/agent/templates/deployment.yaml

```diff
@@ -65,6 +65,10 @@ spec:
             {{- if .Values.dind.enabled }}
             - name: DOCKER_HOST
               value: "tcp://localhost:{{ .Values.dind.port | default "2375" }}"
+            - name: BUILDKITE_BUILD_PATH
+              value: "/var/buildkite/builds"
+            - name: BUILDKITE_PLUGINS_PATH
+              value: "/var/buildkite/plugins"
```

Comment:
So my only probably with setting these (vs. changing the shared_volume to `/buildkite` andd relying on the defaults) is that it seems this change has to be comprehensive (or leave it up to the user).

I mean, what about `BUILDKITE_HOOKS_PATH` - should that be set to `/var/buildkite/hooks` as well? Feels like it should.

Reply:
@0x0I 👍 done